agreement/pseudonode.go

```diff
@@ -33,7 +33,8 @@ import (
 
 // TODO put these in config
 const (
-	pseudonodeVerificationBacklog = 32
+	pseudonodeVerificationBacklog   = 32
+	maxPseudonodeOutputWaitDuration = 2 * time.Second
```

Comment:
Before merging should we consider increasing this constant, or making it dynamic?

Reply:
just thinking out loud: do you think it would be better to separate the agreement changes here into a separate PR ?

Reply:
Yes, thinking about it more now, I agree, keeping agreement changes out of this PR would be a nice way to limit its impact to just required txnsync changes, especially since you fixed the issue leading to the "channel is full" log message separately from the changes in #2582.

Reply:
ok, done.